lib/ProductOpener/Display.pm

```diff
@@ -342,6 +342,37 @@ sub process_template ($template_filename, $template_data_ref, $result_content_re
 	$template_data_ref->{static_subdomain} = $static_subdomain;
 	$template_data_ref->{images_subdomain} = $images_subdomain;
 	$template_data_ref->{formatted_subdomain} = $formatted_subdomain;
+
+	# Read's Release version from version.txt file & Link's the relsease notes of openfoodfacts-bot
+
+	my $version = "";
+	my $anchor = "";
+	if (open my $fh, '<', $BASE_DIRS{RELEASE_VERSION}) {
+		$version = <$fh>;
+		chomp $version;
+		close $fh;
+
+		if ($version){
+			my $owner = 'openfoodfacts';
+			my $repo = 'openfoodfacts-server';
+			$version = 'v' . $version;
+
+			# Construct a custom URL to the release tag
+			my $custom_url = "https://github.com/$owner/$repo/releases/tag/$version";
+
+			# Build the HTML anchor tag with custom display text
+			$anchor = qq{<a href="$custom_url">View Release $version</a>};
```

Comment:
The HTML should ideally be part of the footer template instead of being maintained in Perl. Also, the link text should be localized. Example for a place using both: https://github.com/openfoodfacts/openfoodfacts-server/blob/bea5720420347b16cb7f7b99c3adbfde098d971e/templates/web/common/site_layout.tt.html#L404